exonum-java-binding/core/rust/src/testkit/mod.rs

```diff
@@ -212,6 +220,15 @@ pub extern "system" fn Java_com_exonum_binding_testkit_TestKit_nativeGetEmulated
     unwrap_exc_or(&env, res, JObject::null())
 }
 
+// Deserializes TestKitServiceInstances from its protobuf representation in format ot Java bytes array.
+fn testkit_initialization_data_from_proto(
```

Comment:
nit: testkit_service_instances_from_proto ?

Reply:
Well, it was an initial name, but then I was in doubt since there is info about artifacts as well